packages/starters/backend/sirius-components-task-starter/src/main/java/org/eclipse/sirius/components/task/starter/configuration/AbstractTaskPropertiesConfigurer.java

```diff
@@ -198,16 +201,29 @@ private TextfieldDescription getStartTimeWidget() {
 
     private Function<VariableManager, List<?>> getDependenciesProvider() {
         return variableManager -> {
-            List<Task> dependencies = new ArrayList<>();
-            variableManager.get(VariableManager.SELF, AbstractTask.class)
-                .map(AbstractTask::getDependencies)
+            List<Task> dependencies =  variableManager.get(VariableManager.SELF, AbstractTask.class)
+                .map(this::getProject)
                 .stream()
-                .flatMap(List::stream)
+                .flatMap(this::getAllContentStream)
```

Comment:
can't it be replaced by `.flatMap(project -> project.getOwnedTasks().stream())` ?

Reply:
I need to iterate on the full content inside the Project